neuvector/scan-action
# [NVASHAS-9008] Support sending the scan report to controller

### Summary
- Support Standalone scanning with Jenkins plugin is not sending results to controller

### How to verify
1. add the controller ip, port, username, pwd in the config
2. run the test, make sure it send to cluster
